Payments/Overview/02_Features.md

```diff
@@ -0,0 +1,44 @@
+# Key Features
+Organize and monitor your incoming payments and refunds:   
+
+- [Automated import](./02_Features.md#automated-import)      
+- [Connecting with payment service providers](./02_Features.md#connection-of-payment-service-providers)      
+- [Importing bank statements and automatic posting](./02_Features.md#import-of-bank-statements-and-automatic-posting)   
+- [Payment processing and accounting linked](./02_Features.md#payment-processing-and-accounting-linked)   
+- [Managing all payments centrally](./02_Features.md#managing-all-payments-centrally)  
+- [Handling of sensitive data](./02_Features.md#handling-of-sensitive-data)
+
+<!---Hallo Stefan, die folgenden Infos sind hauptsächlich von der Website. Ich bin mir da nicht sicher, ob das alles so stimmt-->
+
+## Automated import
+With the *Payments* module of Actindo, pre-authorized payments are captured automatically at the moment orders are delivered. If products are announced, but not yet launched, customers can reserve a product and pay later when the shipment has started.
+In case of returns or credit notes, refund requests are automatically sent out to the original means of payment. By using the 4-eyes principle, you can optionally double-check the refunds, for example, with specific user permissions depending on the invoice amount.  
+The single steps of the payments process are usually integrated in an order workflow. The workflow controls that no products are shipped without a payment commitment from the customer.
+
+
+## Connecting with payment service providers
+Youu can flexibly connect various payment service providers such as Adyen&trade;, Paypal&trade; to the *Actindo Core1 Platform*. Depending on the payment service provider you decide for, credit cards, bank transactions, and other standard payment methods are supported implicitly.
+
+
+## Importing bank statements and automatic posting
+Bank statements and reports from payment service providers can be automatically processed and posted to the *Payment Processing* module and therefore by passing the *Payment Processing* module to the Actindo *Accounting* module or a third-party Accounting. In case of prepayments, you can hold the shipment of orders until the invoice amount is received. It is then and automatically released for shipment after payment has been made.
```

Comment:
```suggestion
Bank statements and reports from payment service providers can be automatically processed and posted to the *Payment processing* module and subsequently being passed on to the Actindo *Accounting* module or a third-party accounting system. In case of prepayments, you can hold the shipment of orders until the invoice amount is received. It is then and automatically released for shipment after payment has been made.
```
Diese Info bezieht sich eher, m.E., auf Zahlungsabwicklung, also würde ich wahrscheinlich komplett weglassen. Der letzter Satz (automatically released for shipment after payment has been made) ist vermutlich eine Sache vom Workflow, aber am Besten mit Stefan sprechen.